custom-vu/src/main/kotlin/jces1209/vu/action/CreateAnIssue.kt

```diff
@@ -49,10 +49,22 @@ class CreateAnIssue(
         val driver = jira.driver
         driver
             .wait(
-                condition = elementToBeClickable(createIssueButton),
+                condition = or(*createIssueButtons.map { elementToBeClickable(it) }.toTypedArray()),
                 timeout = Duration.ofSeconds(10)
             )
-            .click()
+        createIssueButtons
+            .forEach(fun(locator) {
+                val foundButtons = driver.findElements(locator)
+                if (!foundButtons.isEmpty()) {
+                    foundButtons
+                        .forEach {
+                            if (it.isDisplayed) {
+                                it.click()
+                                return
+                            }
+                        }
+                }
+            })
```

Comment:
```suggestion
        createIssueButtons
            .flatMap { driver.findElements(it) }
            .filter { it.isDisplayed() }
            .forEach { it.click() }
```

Reply:
Thanks. Still not used to `stream` style